CFC_WebApp/main/tripManager.py

```diff
@@ -72,15 +72,14 @@ def queryUnclassifiedSections(uuid):
 
     # totalUnclassifiedSections are for debugging only, can remove after we know that this works well
     totalUnclassifiedSections=Sections.find({"$and":[ {'source':'Shankari'},
-                                                 {'user_id':user_uuid},
                                                  {'confirmed_mode': ''},
                                                  { 'type': 'move' }]})
 
     unclassifiedSectionCount = unclassifiedSections.count()
     totalUnclassifiedSectionCount = totalUnclassifiedSections.count()
 
-    logging.debug('Unsec.count = %s' % unclassifiedSectionCount)
-    logging.debug('Total Unsec.count = %s' % totalUnclassifiedSectionCount)
+    print('Unsec.count = %s' % unclassifiedSectionCount)
```

Comment:
Please revert change unless you can justify it.
To revert, you can simply do:
git checkout CFC_WebApp/main/tripManager.py